.gitignore

```diff
@@ -136,3 +136,15 @@ dmypy.json
 
 # Cython debug symbols
 cython_debug/
+
+# Editors
+.idea
+[._]*.s[a-v][a-z]
+[._]*.sw[a-p]
+[._]s[a-rt-v][a-z]
+[._]ss[a-gi-z]
+[._]sw[a-p]
+Session.vim
+Sessionx.vim
+*~
+\#*\#
```

Comment:
I like to keep my projects editor-agnostic. So I generally recommend putting this in your personal `~/.gitignore` via `~/.gitconfig`, such as: 

```
[core]
	editor = vim
	excludesfile = ~/.gitignore
```

Reply:
I was wondering if that was the workaround you would suggest :-) Sure, will do it.